Corona-Warn-App/src/main/res/values/colors.xml

```diff
@@ -24,6 +24,8 @@
     <color name="colorTextSemanticGreen">#2E854B</color>
     <color name="colorTextSemanticNeutral">#5D6E80</color>
     <color name="colorTextTint">#007FAD</color>
+    <color name="colorTextGreenButtonPressed">#FFFFFF</color>
```

Comment:
Sorry for not mentioning it before but for the color states we use: https://developer.android.com/guide/topics/resources/color-list-resource 

With that I don't think that this has to added here.

Reply:
And I can't see a green button in the screens.